packages/airnode-validator/README.md

```diff
@@ -32,7 +57,7 @@ specification conversion and validation. Usage of these keywords is described in
 - [optional parameters](docs/optional.md)
 - [nested templates](docs/template.md)
 
-## Documentation
+## Other Documentation
```

Comment:
Perhaps this should be in the monorepo docs only as well, as its instructions for generating the documentation

Reply:
@drgy The section you mention is in the monorepo as you have indicated. Perhaps a better name would be **Source Documentation**? That paragraph is not in the Docs.

Reply:
@wkande 
> The section you mention is in the monorepo as you have indicated

Sorry my bad, I mixed up the repos.

> Do you know why the CI prettier failed?

Try running: `yarn run prettier:write`

Reply:
@drgy 
Sorry I am still confused. Why would we move the section called **Documentation** (in the monorepo) to the Docs? 
Is it not related to template generation? Should it be called **Template Documentation**? If we do move them then why are we not moving the section named **Templates** as well?

Reply:
@wkande I apologize for making things even more confusing. I thought this was the docs repo, I suggested to move the docs generation from api3 docs to airnode readme, after that I noticed that this is the airnode repository, so everything is as it should be, nothing should change, please ignore my suggestion.

Reply:
@drgy 
Ok, then I guess this PR can be merged? I do think that the name **Other Documentation** heading that I had added might be better to rename **Template Documentation**. I can change it if you want. Also I will leave the merge to you since this is your project.